yt/loaders.py

```diff
@@ -1291,6 +1296,11 @@ def load_sample(fn, progressbar: bool = True, timeout=None, **kwargs):
     the data registry, the ones passed to this function take priority.
     """
 
+    if fn is None:
+        print("One can see which sample datasets are available at: https://yt-project.org/data")
+        print("or alternatively by running: yt.sample_data.api.get_data_registry_table()")
+        return
```

Comment:
The print statements should be merged and print to sys.sidéré
The return statement should be an explicit None

Reply:
I thought it was generally bad form to have code lines that extended beyond 80 characters.  Having these lines merged would do so.

I don't know what sys.sidéré is.

Reply:
> I thought it was generally bad form to have code lines that extended beyond 80 characters. Having these lines merged would do so.

our limit is 88, and there is a way to merge strings without going over it :)

> I don't know what sys.sidéré is.
that what I get for reviewing on my phone...

Reply:
Hahahhaha...:)